sycl/doc/extensions/experimental/sycl_ext_oneapi_properties.asciidoc

```diff
@@ -405,6 +405,8 @@ class properties {
   static constexpr auto get_property();
 };
 
+using empty_property_list = decltype(properties{});
+
```

Comment:
Could you also add a sentence below describing this type alias?  For example, add this right after the table of member functions:

> The `empty_properties_t` type alias is a shorthand for the type of an empty property list.